mcs/tools/mdoc/Mono.Documentation/assembler.cs

```diff
@@ -76,22 +82,30 @@ public override void Run (IEnumerable<string> args)
 			return;
 
 		List<Provider> list = new List<Provider> ();
-		EcmaProvider ecma = null;
+		List<EcmaProvider> ecma = null;
```

Comment:
Ahh, you know what ... this is an artifact from an earlier implementation that I was going for. I eventually changed how I was going to do multiple runs (because I needed to be able to do everything, for both classic and unified assemblies in one invocation, but it's much easier to keep the logic separate while it's running ... so I just run it twice internally for each set of assemblies).

I can change this and will resubmit